e2e/preview-panel/preview-panel.e2e-spec.ts

```diff
@@ -37,8 +37,6 @@ describe('Preview panel', function() {
     const middleColumn = element(by.id('middle-main-container'));
     const rightColumn = element(by.id('right-main-container'));
 
-    expect(previewToggleButton.getText()).toEqual('Hide Preview');
```

Comment:
also you need remove the line where `previewToggleButton` is selected

Reply:
I think I would have to keep this line because later on the test "clicks" the `previewToggleIcon` (which is the id now given to a span element instead of a button) to check that the sizes of the containers change after the click.